### PR TITLE
Warning Message Fixes

### PR DESCRIPTION
**Pull request by nicbiggs**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -9,7 +9,7 @@ project(Server)
 enable_testing()
 add_test(NAME UnitTests COMMAND Test)
 set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_LIST_DIR}) # Findexternpro.cmake
-set(externpro_REV 16.10.3)
+set(externpro_REV 16.07.1)
```
Comment (katiesweet) [inline CMakeLists.txt]: A lot of people are running 16.10.3. Anyone using Sundberg's externpro build would need to change this locally before their server will run if we keep this change.

**Comment by C-Merrill** [comment]
@Dakron15 Katie is right on that review. We've updated to version 10 so this represents regression.

**Comment by nicbiggs** [comment]
Sorry that was my fault. I've just been using the previous version but I forgot to remove that before sending the pull request. Thanks for catching that!

**Comment by C-Merrill** [comment]
Looks like you got rid of all warnings but one. @davidbrowning you may want to review this function.
capture

**Comment by mikamerath** [comment]
@C-Merrill, I made a separate issue for the spades logic recursion, As long as this builds on Linux, I would be fine merging this in. @katiesweet, can you test this?

**Review by mikamerath — APPROVED** [review]

**Comment by davidbrowning** [comment]
@C-Merrill beginRound was removed yesterday as implementing the PlayerAPI rendered it obsolete.

**Comment by katiesweet** [comment]
I get these build errors.

screen shot 2016-12-02 at 5 23 15 pm

**Comment by nicbiggs** [comment]
That's interesting, I wasn't getting those errors but I can look into it more tonight to see what's up.

On Fri, Dec 02, 2016 at 5:24pm, katiesweet < notifications@github.com [notifications@github.com] > wrote:
I get these build errors.

[https://cloud.githubusercontent.com/assets/16142858/20854805/1b32b552-b8b4-11e6-8b8b-2a5281adffa9.png]

**Comment by C-Merrill** [comment]
@katiesweet perhaps including &lt;time.h&gt; will get fix those build errors. I would push a commit that does it, but I can't push to this particular fork/branch. @Dakron15 will you merge from master and resolve any other build warnings that exist after the merge?

**Comment by MadCook** [comment]
Katie has gone to bed.

**Comment by nicbiggs** [comment]
Yes I can do that. From what I can tell the localtime_s doesn't have Linux support and also is supposedly not used much anymore. The original warning recommended it over the "regular" localtime so we might need to find another way if adding <time.h> doesn't work.

On Sat, Dec 03, 2016 at 1:08am, Chris T Merrill < notifications@github.com [notifications@github.com] > wrote:
@katiesweet [https://github.com/katiesweet] perhaps including <time.h> will get fix those build errors. I would push a commit that does it, but I can't push to this particular fork/branch. @Dakron15 [https://github.com/Dakron15] will you merge from master and resolve any other build warnings that exist after the merge?

**Comment by katiesweet** [comment]
Sorry all, I got a splitting headache last night and needed to try and sleep it off. 

@Dakron15 Still fails, even with <time.h>. I think the problem is this (from Stack Overflow):
localtime_s is just a microsoft implementation of the localtime functon, you can safely keep using locatime becaue it's C++ ISO compliant and ony microsoft marked it as "deprecated". The localtime function itself isn't deprecated at all in the C++ world.

**Comment by nicbiggs** [comment]
That's interesting. Well I'd be fine putting it back to being localtime as long as we're ok with the warning message visual studio gives (since Microsoft is the only one complaining about it).

**Comment by C-Merrill** [comment]
@Dakron15 change those 2 things back, @katiesweet check the build and lets get this going.

**Comment by C-Merrill** [comment]
It appears there's still one bool int mismatch in a test case.

**Comment by C-Merrill** [comment]
I'll pull this in and fix the last one.

**Pull request merged**